apps/minifront/src/components/shared/input-token.tsx

```diff
@@ -17,6 +17,7 @@ interface InputTokenProps extends InputProps {
   setSelection: (selection: BalancesResponse) => void;
   validations?: Validation[];
   balances: BalancesResponse[];
+  onChange?: React.ChangeEventHandler<HTMLInputElement>;
```

Comment:
In the interest of not letting consumers mess with this component too much (via `className`/etc.), this limits the props to only the ones we want to allow consumers to set.